### PR TITLE
 multimc: create compatibility symlink to fix desktop icon (18.03)

### LINKED ISSUE #42784: multimc does not create a system menu entry/.desktop file
## Issue description

The multimc package does not install a .desktop file, causing it to not show up in the system application launcher.

### Steps to reproduce

* Install multimc (nix-env -i multimc)
* Logout or reboot
* Search through the system launcher menu. No MultiMC entry will be present.
* Verify that multimc is installed by running `MultiMC` in the terminal.

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.51, NixOS, 18.03.132797.0a70d65fd47 (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0.4`
 - channels(root): `"nixos-18.03.132304.45cd6090d9a"`
 - channels(amelorate): `"nixos-18.03.132304.45cd6090d9a, nixos-unstable-18.09pre139651.6db7f92cc2a"`
 - nixpkgs: `/nix/store/nlyhbhcfbsjmzysgzrpqm4hqlf0krpqh-nixexprs.tar.xz`

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
close #42784 for 18.03
This also includes an update -> 0.6.2 which I thought would be safe to do here also.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/multimc/default.nix [hunk 1]
```diff
@@ -18,7 +18,16 @@ in stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   postInstall = ''
+    mkdir -p $out/share/{applications,pixmaps}
+    cp ../application/resources/multimc/scalable/multimc.svg $out/share/pixmaps
```
Comment (Infinisil) [inline pkgs/games/multimc/default.nix]: What is $out/share/pixmaps for? Am curious

**Comment by Infinisil** [comment]
@GrahamcOfBorg build multimc

**Comment by Infinisil** [comment]
This is weird.. Why did it succeed on x86, but fail with the unfree message on aarch..

File: pkgs/games/multimc/default.nix (hunk 1 above)
Comment (worldofpeace) [reply pkgs/games/multimc/default.nix]: Stuff for like apps like this with pixmaps... with java? Idk :joy:

**Comment by samueldr** [comment]
```
Package ‘oraclejdk-8u171’ in /var/lib/gc-of-borg/nix-test-rs-26/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/grahamc-aarch64-community-26/pkgs/development/compilers/oraclejdk/jdk-linux-base.nix:71 has an unfree license (‘unfree’), refusing to evaluate.
```

The JDK used @Infinisil

**Comment by Infinisil** [comment]
Ah right, will check the logs next time. Didn't realize that it's possible to have different derivations depending on the arch.

**Pull request merged**